tests/resource_filtering_test.go

```diff
@@ -1527,30 +1498,21 @@ var _ = Describe("Resource includes", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Veryfing backup")
-				backup, err := framework.GetBackup(timeout, backupName, f.BackupNamespace)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(backup.Status.Progress.ItemsBackedUp).To(Equal(backup.Status.Progress.TotalItems))
-
-				// The backup should contains the following 13 items:
-				// - VirtualMachineInstance
-				// - Launcher pod
-				// - DataVolume
-				// - DV's PVC
-				// - DV's PVC's PV
-				// - standalone PVC
-				// - standaolne PVC's PV
-				// - VolumeSnapshot (DV)
-				// - VolumeSnapshotContent (DV)
-				// - VolumeSnapshot (PVC)
-				// - VolumeSnapshotContent (PVC)
-				// - VolumeSpapshotClass
-				// - DataVolume CRD
-				// - Namespace
-				expectedItems := 14
-				if framework.IsDataVolumeGC(f.KvClient) {
-					expectedItems -= 1
+				expectedItems := map[string][]string{
+					"CustomResourceDefinition": []string{"virtualmachineinstances"},
+					"VirtualMachineInstance":   []string{vmi.Name},
+					"Pod":                      []string{"virt-launcher"},
+					"DataVolume":               []string{dvName},
+					"PersistentVolume":         []string{"pvc"},
```

Comment:
Not crazy about these PV checks, there is guarantee that the PV name will contain "pvc" also I think it would be better if  `checkBackupResources` does an exact match (but not sure what to do about the Pod check).  Maybe have a `checkPVC` function that checks the PVC then gets the volumename from the PVC and looks up the PV?

Reply:
yeah I get what you are saying about the pv check, I didn't like it either lol but didn't want to over complicate stuff, I guess I can get the pv name from the pvc and check for it..
regarding the virt-launcher pod I can add to the name "virt-launcher"-vm.name I think that will be specific enough what do you think?

Reply:
done